library/tiqr/Tiqr/Message/FCM.php

```diff
@@ -92,6 +93,13 @@ private function _sendFirebase($deviceToken, $alert, $challenge, $apiKey)
             throw new Tiqr_Message_Exception_SendFailure("Http error occurred: ". $errors, true);
         }
 
+        // Wait and retry once in case of a 502 Bad Gateway error
+        if (($statusCode == 502) && !($retry)) {
```

Comment:
Nitpicks: 
1. You can be even more strict in this check by using `===` that way type equality is also tested.
2. The extra sets of parentheses are not necessary and only improve readability marginally. Consider removing them from the status code check and the retry portion of the expression.